src/engine/source/base/src/utils/timeUtils.cpp

```diff
@@ -29,9 +29,14 @@ std::string getTimestamp(const std::time_t& time, const bool utc = true)
     ss << std::setfill('0') << std::setw(2) << std::to_string(localTime->tm_sec);
     return ss.str();
 }
-std::string getCurrentTimestamp()
+std::string getCurrentTimestamp(const bool onlyDate = false)
 {
-    return getTimestamp(std::time(nullptr));
+    std::string currentTimestamp {getTimestamp(std::time(nullptr))};
+    if (onlyDate)
+    {
+        currentTimestamp = base::utils::string::split(currentTimestamp, ' ').at(0);
+    }
+    return currentTimestamp;
 }
```

Comment:
Maybe we can create another method named getCurrentDate, just to simplify the function usage

This:
```C++
auto date = getCurrentDate();
```

Is much more transparent that:
```C++
auto date = getCurrentTimestamp(true);
```

----
Also, we can make the function to return the proper separator 

```C++
std::string getCurrentDate(const std::string_view separator = "/")
{
   auto date = base::utils::string::split(getCurrentTimestamp(), ' ').at(0);
   Utils::replaceAll(date, "/", separator)
   return date;
}
```

Reply:
Done